### PR TITLE
[docs] Update configurations for readthedocs

### PR DESCRIPTION
**Pull request by VSevagen**
Updated configuration for readthedocs to have
up-to-date perceval documentation.

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

File: docs/howto_doc.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# How to generate documentation locally.
```
Comment (vchrombie) [inline docs/howto_doc.md:1]: Do you think a `README.md` file inside the `docs/` folder would be a good idea?

We can transfer this content to the readme.

**Review by vchrombie — COMMENTED** [review]
Thanks for opening the PR @VSevagen.
I understand this PR would be aiming at fixing the broken docs.

Do you think we can remove the other files in the [docs](https://github.com/chaoss/grimoirelab-perceval/tree/master/docs) folder?
Maybe change the place where they should be?

File: docs/howto_doc.md (hunk 1 above)
Comment (VSevagen) [reply docs/howto_doc.md:1]: Sure. We can do that. But if we do that there will be 2 files with the same data. If we have the README.md file, I think we can omit the howto_doc.md file. That works out ?

**Comment by vchrombie** [comment]
Would be interested to listen to other's comments as well.

The [sortinghat docs](https://sortinghat.readthedocs.io/) is now running using poetry. We would like to update the perceval docs in the same way [sortinghat/docs](https://github.com/chaoss/grimoirelab-sortinghat/tree/muggle/docs). A webhook will be created in the readthedocs project, so each time something is pushed the documentation will be built and updated.

@sduenas @jgbarah WDYT?

**Comment by VSevagen** [comment]
> Thanks for opening the PR @VSevagen.
> I understand this PR would be aiming at fixing the broken docs.
> 
> Do you think we can remove the other files in the [docs](https://github.com/chaoss/grimoirelab-perceval/tree/master/docs) folder?
> Maybe change the place where they should be?

We could remove them if they don't serve any purpose. I'm not sure about changing the place since only sphinx understand `.rst` format and can make something out of it.

**Comment by vchrombie** [comment]
This PR would address the documentation outdated issues like https://github.com/chaoss/grimoirelab-perceval/issues/625.

**Review by vchrombie — CHANGES_REQUESTED** [review]
Hi @VSevagen, I have some more suggestions and after that, we can move forward to merge the PR.

Can you update the conf.py file so that it matches the [pyproject.toml](https://github.com/chaoss/grimoirelab-perceval/blob/master/pyproject.toml)?
```
project = 'Perceval'
- copyright = '2015-2016, Bitergia'
+ copyright = '2021, GrimoireLab'
- author = 'Bitergia'
+ author = 'GrimoireLab Developers'
```

**Comment by VSevagen** [comment]
@vchrombie , I updated the PR.

**Review by vchrombie — APPROVED** [review]
LGTM!
Thanks for your contribution @VSevagen.

**Comment by sduenas** [comment]
I think you should update the documents to include the right copyright. It should have the original from Bitergia and add the new one for GrimoireLab developers for this year. I think this is the right way because part of the documentation is generated from the code documentation and originally had that copyright.

**Comment by vchrombie** [comment]
> I think you should update the documents to include the right copyright. It should have the original from Bitergia and add the new one for GrimoireLab developers for this year. I think this is the right way because part of the documentation is generated from the code documentation and originally had that copyright.

Sorry @sduenas, but I think I didn't understand.

Right now, the documentation has `Copyright © 2021, GrimoireLab`
https://github.com/chaoss/grimoirelab-perceval/blob/639b8cada30214f2e5a6867f1b95c3d155ce4258/docs/conf.py#L26

Should we revert this change?

> Can you update the conf.py file so that it matches the [pyproject.toml](https://github.com/chaoss/grimoirelab-perceval/blob/master/pyproject.toml)?
> 
> ```
> project = 'Perceval'
> - copyright = '2015-2016, Bitergia'
> + copyright = '2021, GrimoireLab'
> - author = 'Bitergia'
> + author = 'GrimoireLab Developers'
> ```

Thanks.

**Comment by sduenas** [comment]
It should be something like:

```
2015-2021, Bitergia
2021, Grimoirelab Developers
```

**Comment by vchrombie** [comment]
> It should be something like:
> 
> ```
> 2015-2021, Bitergia
> 2021, Grimoirelab Developers
> ```

Okay, that looks good. I will open a PR clarifying the change.

**Pull request merged**